cmd/transfer-pvc/transfer-pvc.go

```diff
@@ -370,6 +427,80 @@ func followClientLogs(srcConfig *rest.Config, c client.Client, namespace string,
 	return err
 }
 
+func createAndWaitForIngress(pvc *corev1.PersistentVolumeClaim, destClient client.Client) endpoint.Endpoint {
+	// create a route for data transfer
+	// TODO: add a config flag for subdomain
```

Comment:
Same as below. Should the subdomain be accepted as an argument?
Taking care of this in this PR or follow-up?

Reply:
the PR that add the flag should do that IMO, do you mind if I create an issue for this to be taken care of as a follow up?

Reply:
works for me. add that as an issue.